web/src/components/common/Button.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import MuiButton from 'muicss/lib/react/button';
+import styled from 'styled-components';
+
+const BUTTON_FONT_COLORS: {[key: string]: string} = {
+  'bright-red': 'white',
+  'dark-gray': 'white',
+  green: 'white',
+  'pale-green': 'var(--green)',
+  'pale-gray': 'var(--gray)',
+};
+
+/**
+ * Button with dark gray background, white uppercase fonts and rounder corners.
+ */
+const Button = styled(MuiButton)`
```

Comment:
Do we only have one kind of button? If not maybe give it a more concrete name

Reply:
@chshda Yep! So far there is only one generic button. But I guess we can name it something like `RoundedButton` to make it more specific.